packages/dx-react-grid/src/plugins/table-edit-row.jsx

```diff
@@ -36,8 +36,11 @@ export class TableEditRow extends React.PureComponent {
           predicate={({ tableRow, tableColumn }) => isEditExistingTableCell(tableRow, tableColumn)}
           connectGetters={(getter, { tableColumn: { column }, tableRow: { rowId, row } }) => {
             const change = getRowChange(getter('changedRows'), rowId);
+            const getCellData = getter('getCellData');
+
             return {
-              value: column.name in change ? change[column.name] : row[column.name],
+              value: getCellData(change, column.name) || getCellData(row, column.name),
```

Comment:
if change = { user: undefined } and row = { user: 'A user' }, then the value will be 'A user' but should be undefined.